<launch>
  <node name="id_solver_node"
        ns="collision_detection"
        pkg="collision_detection"
        type="solver_node_wbc"
        output="screen">
    <remap from="dynamics_joint_states" to="joint_states"/>
    <rosparam command="load" file="$(find collision_detection)/config/pr2_l_arm.yaml" />
    <param name="/sai_xml" value="$(find collision_detection)/config/pr2_left_arm_wbc.sai.xml"/>
  </node>
</launch>

